scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/util/ChangeDBConfiguration.java

```diff
@@ -0,0 +1,686 @@
+        } else {
+            exitWithErrorMessage("--" + USERNAME_OPTION_NAME + " is required", null, null);
+        }
+        if (cmd.hasOption(PASSWORD_OPTION_NAME)) {
+            password = cmd.getOptionValue(PASSWORD_OPTION_NAME).trim();
+            if (!password.startsWith("ENC(")) {
+                password = "ENC(" + PropertyDecrypter.getDefaultEncryptor().encrypt(password) + ")";
+            }
+        } else {
+            exitWithErrorMessage("--" + PASSWORD_OPTION_NAME + " is required", null, null);
+        }
+        if (cmd.hasOption(DB_HOST_OPTION_NAME)) {
+            if (url != null) {
+                exitWithErrorMessage("When --" + URL_OPTION_NAME + " is provided, the -- " + DB_HOST_OPTION_NAME +
+                                     " option cannot be used", null, null);
+            }
+            hostname = cmd.getOptionValue(DB_HOST_OPTION_NAME).trim();
+            Matcher hostnameMatcher = hostnamePattern.matcher(hostname);
+            Matcher ipMatcher = ipPattern.matcher(hostname);
+            if (!hostnameMatcher.find() && !ipMatcher.find()) {
+                exitWithErrorMessage(DB_HOST_OPTION_NAME + " must be either a valid hostname or ip address.",
+                                     null,
+                                     null);
+            }
+        } else if (url == null) {
+            hostname = "localhost";
+        }
+        if (cmd.hasOption(DB_PORT_OPTION_NAME)) {
+            if (url != null) {
+                exitWithErrorMessage("When --" + URL_OPTION_NAME + " is provided, the --" + DB_PORT_OPTION +
+                                     " option cannot be used", null, null);
+            }
+            port = cmd.getOptionValue(DB_PORT_OPTION_NAME).trim();
+            try {
+                int portValue = Integer.parseInt(port);
+                if (portValue < 0 || portValue > 65353) {
+                    throw new IllegalArgumentException();
+                }
+                if (portValue < 1024 && vendor.equals("hsqldb")) {
+                    System.err.println("WARN: port range from 0 to 1023 are accessible only to privileged users. Using it within ProActive require additional operating system configuration.");
+                }
+            } catch (Exception e) {
+                exitWithErrorMessage(DB_PORT_OPTION_NAME + " must be a valid port value.", null, null);
+            }
+        } else if (url == null) {
+            port = defaultPorts.get(vendor);
+        }
+        if (cmd.hasOption(MYSQL_TIMEZONE_OPTION_NAME)) {
+            if (url != null) {
+                exitWithErrorMessage("When --" + URL_OPTION_NAME + " is provided, the --" + MYSQL_TIMEZONE_OPTION_NAME +
+                                     " option cannot be used", null, null);
+            }
+            timezone = cmd.getOptionValue(MYSQL_TIMEZONE_OPTION_NAME).trim();
+        } else if (url == null) {
+            timezone = "UTC";
+        }
+
```

Comment:
either add line break everywhere between Ifs or not

Reply:
I added line breaks as suggested